src/Admin/AbstractAdmin.php

```diff
@@ -1866,7 +1866,7 @@ public function setUniqid($uniqid)
     public function getUniqid()
     {
         if (!$this->uniqid) {
-            $this->uniqid = 's'.uniqid();
+            $this->uniqid = 's'.crc32($this->getBaseCodeRoute());
```

Comment:
I wonder what this s is for

Reply:
This name is slightly misleading now

Reply:
It was added because fields ids can't start with a number

Reply:
> This name is slightly misleading now

Probably not a big deal though